_newsletters/2021-07-01-july-2021.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: July 2021 Newsletter
+category: newsletters
+last_modified_at: 
+tags: newsletters
+newsletter-year: 2021
+sidenav: about_us
+---
+
+## Dispatch from the Help Desk
+### Question: Can pages be removed from our search results?
+
+This question has two answers, depending on which index supports your search experience.
+
+*   If your search site uses our index, we scan for page updates and removals on a regular basis. If you need something removed immediately, please email us. You can also use the [Filter URLs](https://search.gov/manual/filter-content.html) feature to hide the item.
+*   If your search site uses the Bing index, you can use the Filter URLs feature, but we’re unable to update their index directly. Often, we resolve the issue by indexing your site and moving it onto our index, where we can have control over the content and timeliness of updates.
+
+Learn more about the [Filter URLs feature](https://search.gov/manual/filter-content.html).
+
+## SEO Pro Tip for Drupal Users
+### Simple Sitemap vs. XML Sitemap Module
+
+As more teams are working on site redesigns and upgrades, we wanted to alert Drupal users to some important differences between the available XML sitemap modules. In order to support search engine indexing processes, all the files that you want to appear in search results should be listed in the XML sitemap. This includes PDFs, docs, etc. If you’re new to XML sitemaps, please [read our explainer page](https://search.gov/manual/sitemaps.html).
+
+In the Simple XML Sitemap module, file attachments need to be added to the XML sitemap manually, one by one. For many teams, this is not a sustainable way to ensure PDFs appear in search results.
+
+We recommend that all Drupal sites use the [XML Sitemap module](https://www.drupal.org/project/xmlsitemap), if possible: it has a similar name, but different features. This module allows you to include file attachments globally by type, with a simple check box within the module settings, via the Sitemap Entities tab.
+
+![Screen Shot 2021-01-29 at 10.46.21 AM](https://cdn2.hubspot.net/hub/1962994/hubfs/Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png?upscale=true&width=1120&upscale=true&name=Screen%20Shot%202021-01-29%20at%2010.46.21%20AM.png)
```

Comment:
Do we have a native version of this?

Reply:
don't want the image reference to break. so we should either create a version of the file on our site, or remove from the newsletter.